holmes/core/subagent.py

```
@@ -0,0 +1,192 @@
+            description="A short (3-5 word) label describing the subtask, used for logging.",
+        ),
+        "prompt": ToolParameter(
+            type="string",
+            required=True,
+            description=(
+                "The full task prompt for the subagent. Must be self-contained "
+                "because the subagent does NOT see your conversation history. "
+                "Include all relevant context, constraints, and what you want back."
+            ),
+        ),
+    }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)